.ci_support/linux_64_c_compiler_version7cuda_compiler_version10.2cxx_compiler_version7.yaml

```diff
@@ -3,7 +3,7 @@ c_compiler:
 c_compiler_version:
 - '7'
 cdt_name:
-- cos7
+- cos6
```

Comment:
Hi @jakirkham this change is a side effect of me removing all `linux64` related stuff from cbc.yaml (which is still needed for aarch64/ppc64le), would this be a concern?

Reply:
Fixed in #24.

Reply:
Sorry for missing this. Looks like this was addressed in PR ( https://github.com/conda-forge/cutensor-feedstock/pull/24 )?

Mainly this impacts CDTs, which are low-level system packages on Linux that may be needed at build and/or test time (think graphics renderers, printer drivers, etc.). This package doesn't seem to be using those.

Reply:
Yup thanks John, #24 fixed it, in the sense that setting to cos7 would enable some selectors like newer sysroot in the recipe, which cuTENSOR requires (glibc >=2.17 IIRC). Otherwise there's no CDT involved like you said.